src/tsconfig-base.json

```diff
@@ -4,8 +4,8 @@
         "outDir": "../built/local",
 
         "pretty": true,
-        "lib": ["es2018"],
-        "target": "es2018",
+        "lib": ["es2020"],
+        "target": "es2020",

```

Comment:
Perhaps this should be set to ES2019 to be safe about the Node 14 problems.

Reply:
I set these to es2019 in an abundance of caution.

Reply:
Though, that feels inconsistent :(

Reply:
I just flipped it back. We'll k now if we have to do it but since we aren't using this output anyway I don't think the inconsistency is a good idea.